---
tags: ,error-messages
title: "Should mention callback_context in the error message about combining multiple outputs into a single callback"
html_url: "https://github.com/plotly/dash/issues/1141"
user: chriddyp
repo: plotly/dash
---

```
dash.exceptions.DuplicateCallbackOutput: 
You have already assigned a callback to the output
with ID "column-graph" and property "figure". An output can only have
a single callback function. Try combining your inputs and
callback functions together into one function.
```

Users run into this issue for two main reasons:
1. They copied and pasted a previous callback, and now need to update the IDs
2. They want to update an output conditionally based off a certain input changing

For 2, they need to use `callback_context` and `triggered`. We should tell them that in this error message.